RLTSQUARE/Unit3/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="rltsquare" translate="label" sortOrder="200">
+            <label>RLTSquare</label>
+        </tab>
+        <section id="unit3" translate="label" type="text" sortOrder="42" showInDefault="1"
+                 showInWebsite="1" showInStore="1">
+            <label>Unit 3</label>
+            <tab>rltsquare</tab>
+            <resource>RLTSquare_Unit3::config</resource>
+            <group id="general" translate="label" sortOrder="92" showInDefault="1" showInWebsite="1" showInStore="1">
+                <label>Settings</label>
+                <field id="username" translate="label" type="text" sortOrder="10" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>User Name</label>
+                </field>
+                <field id="password" type="obscure" translate="label" sortOrder="20" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Password and username fields should have been set for each environment ... like staging_password, production_password ... then use the to hide/show it based of environment type. We want it so that when we switch environment, we have the user/pass stored specific to each env. The helper functions such as getPassword can then just return pass based of chosen env.